koku/api/report/serializers.py

```diff
@@ -317,7 +317,17 @@ class ParamSerializer(BaseSerializer):
     currency = serializers.ChoiceField(choices=CURRENCY_CHOICES, required=False)
     category = StringOrListField(child=serializers.CharField(), required=False)
 
-    order_by_allowlist = ("cost", "supplementary", "infrastructure", "delta", "usage", "request", "limit", "capacity")
+    order_by_allowlist = (
```

Comment:
This could be a `set` based on how it is used. Also might be nice to alphabetize this to make it easier on the reader since order doesn't matter to the program.